app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventFragment.java

```diff
@@ -156,9 +155,9 @@ public void showProgress(boolean show) {
 
     @Override
     public void onSuccess(String message) {
-        Intent intent = new Intent();
-        intent.putExtra("message", message);
-        getActivity().setResult(Activity.RESULT_OK, intent);
+        final int duration = Toast.LENGTH_SHORT;
+        Toast toast = Toast.makeText(getActivity().getApplicationContext(), message, duration);
+        toast.show();
```

Comment:
Since we do not want a reference of the toast to work on, we can just do `Toast.makeText(getActivity(), message, Toast.LENGTH_SHORT).show()`

Also, no need to get Application Context